app.yaml

```diff
@@ -36,3 +36,14 @@ automatic_scaling:
 
 env_variables:
   NODE_ENV: production
+
+# Temporary workaround for a Cloud SDK bug.
+# Ensures that node_modules directory is not uploaded (the other entries are the
+# default values for skip_files).
+skip_files:
```

Comment:
My thinking was that we wanted the exact same behavior as we have currently, but with the addition of the `node_modules` directory, so I took the [default skip_files setting](https://cloud.google.com/appengine/docs/python/config/appconfig?hl=en#Python_app_yaml_Skipping_files) and added the `node_modules` line. You have to do that in order to add a `skip_files` entry, rather than replace the `skip_files` value:

> If a new value is specified in app.yaml, it overrides the default value. To extend the above regular express list, copy and paste the above list into your app.yaml and add your own regular expressions.

I'm happy to take everything else out if you prefer, but I think this will prevent e.g. editor files from getting uploaded.